helm/cluster-apps-operator/values.yaml

```diff
@@ -1,4 +1,4 @@
-baseDomain: ghost.westeurope.azure.gigantic.io
+base: ginger.eu-west-1.aws.gigantic.io
```

Comment:
This is the fix.

I forgot to commit this in the last PR 🤦‍♂️  and it caused the operator to crash.